customHeaders.go

```diff
@@ -17,9 +18,22 @@ type HeaderConfigArray struct {
 
 // HeaderConfig is a single header rule specification
 type HeaderConfig struct {
+	Regex         string            `json:"regex"`
 	Path          string            `json:"path"`
 	FileExtension string            `json:"fileExtension"`
 	Headers       []HeaderDefiniton `json:"headers"`
+
+	CompiledRegex *regexp.Regexp
+}
+
+func (config *HeaderConfig) Init() {
+	if len(config.Regex) > 0 {
+		config.CompiledRegex = regexp.MustCompile(config.Regex)
```

Comment:
`MustCompile` is firing a panic, can you return an error?

Reply:
@PierreZ, sure thing. I changed it so that it will print a Warning and discard the invalid rule if the regex is invalid.